src/services/FacilityService.ts

```diff
@@ -116,7 +123,18 @@ const getFacilities = async ({ type, organizationId, query }: FacilitySearchPara
 
   const params: SearchNodePayload = {
     prefix: 'facilities',
-    fields: ['id', 'name', 'description', 'type', 'organization_id', 'timeZone'],
+    fields: [
+      'id',
+      'name',
+      'description',
+      'type',
+      'organization_id',
+      'timeZone',
+      'buildStartedDate',
+      'buildCompletedDate',
+      'operationStartedDate',
+      'capacity',
```

Comment:
these search fields used primarily in the reports section?

Reply:
I included them here because they are a general property of the facility now.